applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -575,58 +577,74 @@ void UPwSmallStrainInterfaceElement<TDim,TNumNodes>::
                                  const ProcessInfo& rCurrentProcessInfo)
 {
     KRATOS_TRY;
+
+    const GeometryType& Geom = this->GetGeometry();
+    const unsigned int NumGPoints = Geom.IntegrationPointsNumber(mThisIntegrationMethod);
+    std::vector<double> GPValues(NumGPoints);
+
+    //Printed on standard GiD Gauss points
+    const unsigned int OutputGPoints = Geom.IntegrationPointsNumber(this->GetIntegrationMethod());
+    if (rValues.size() != OutputGPoints)
+        rValues.resize(OutputGPoints);
+
     // KRATOS_INFO("0-UPwSmallStrainInterfaceElement:::CalculateOnIntegrationPoints<double>()") << std::endl;
+    if (rVariable == VON_MISES_STRESS) {
+        //Loop over integration points
+        for (unsigned int GPoint = 0; GPoint < NumGPoints; ++GPoint) {
+            StressStrainUtilities EquivalentStress;
+            GPValues[GPoint] = EquivalentStress.CalculateVonMisesStress(mStressVector[GPoint]);
+        }
 
-    if (rVariable == DAMAGE_VARIABLE) {
+        this->InterpolateOutputDoubles(rValues, GPValues);
+    } else if (rVariable == MEAN_EFFECTIVE_STRESS) {
+        //Loop over integration points
+        for (unsigned int GPoint = 0; GPoint < NumGPoints; ++GPoint) {
+            StressStrainUtilities EquivalentStress;
+            GPValues[GPoint] = EquivalentStress.CalculateMeanStress(mStressVector[GPoint]);
+        }
+    
+        this->InterpolateOutputDoubles(rValues, GPValues);
+    } else if (rVariable == MEAN_STRESS ||
```

Comment:
MEAN_STRESS is the summation of the effective mean stress and pore water pressure. It is not not always zero.

Reply:
I will add it